tests/ops/integration_tests/saas/test_auth0_task.py

```diff
@@ -0,0 +1,165 @@
+import random
+
+import pytest
+import requests
+
+from fidesops.core.config import config
+from fidesops.graph.graph import DatasetGraph
+from fidesops.models.privacy_request import PrivacyRequest
+from fidesops.schemas.redis_cache import PrivacyRequestIdentity
+from fidesops.task import graph_task
+from fidesops.task.graph_task import get_cached_data_for_erasures
+from tests.ops.graph.graph_test_util import assert_rows_match
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_auth0
+def test_auth0_access_request_task(
+    db,
+    policy,
+    auth0_connection_config,
+    auth0_dataset_config,
+    auth0_identity_email,
+) -> None:
+    """Full access request based on the Auth0 SaaS config"""
+    privacy_request = PrivacyRequest(
+        id=f"test_saas_access_request_task_{random.randint(0, 1000)}"
```

Comment:
Let's change the id to `test_auth0_access_request_task` so we can uniquely identify it.

Reply:
> Good first round, I just have some questions around a few things before I can approve. It also looks like the `user_logs` endpoint isn't returning any data at the moment.
`user_logs` are deleted after some days which is why it returns empty. We can verify it by changing password of identity user, then we will get user logs.